src/main/java/io/github/moulberry/notenoughupdates/util/Calculator.java

```diff
@@ -29,6 +29,7 @@
 
 public class Calculator {
 	public static BigDecimal calculate(String source) throws CalculatorException {
+		source = source.toLowerCase();
```

Comment:
`toLowerCase(Locale.ROOT)` is better because it wont have problems with turkish computers or other languages due to lowercase chars being different and wont properly check things like its supposed to.

Reply:
![image](https://user-images.githubusercontent.com/24389977/186992054-c6535ecf-dab2-4871-84a0-fda7f5568f7e.png)
You want to change this for all occurrences in neu in another pr, maybe?

Reply:
well the reason I say it here is that it is possible that in this case it will just not calculate correctly as there wont be a matching char idk about other locations where it is used but in this location specifically it just will output the wrong thing in a different language that changes a lowercase char.

Reply:
but a audit of usages will likely need to be done as yes those other locations could cause a crash or unexpected thing to happen.

Reply:
In the case of this pr it doesn't make sense to support "different" languages. You have to type a Arabic m for million like you need to in the hypixel signs. Extra checks suggests to the user that his format is fine when it's no proper m for hypixel too.
Additionally, adding the lowercase method to the calculator class doesn't break anything for Turkish numbers since it wasn't working before for them anyways